README.md

```diff
@@ -37,3 +37,29 @@ Clear cache:
 ```bash
 bin/console cache:clear
 ```
+
+## Extension points
+
+Customer Reorder plugin is based on two processes:
+* ReorderProcessing
+* ReorderEligibilityChecking
+
+They are both based on Symfony's compiler passes and configured in `services.xml` file.
+
+ReorderProcessing and EligibilityChecking are independent processes - once a Reorder
+is created using Processors (services tagged as `sylius.reorder_processor`), the created
```

Comment:
I know that it is only docs to the implemented features but.. shouldn't we use `sylius_customer_reorder_plugin` (instead of `sylius`) as a prefix to these tags? Wdyt? /cc @bartoszpietrzak1994 @Zales0123